docs/maintainers/niche-package-maintenance/rustc/rust-packaging-faq.md

```diff
@@ -0,0 +1,263 @@
+
+`dch`
+: Automation for managing the `/debian/changelog` file.
+It helps you generate timestamps, author information, and more.
+
+`git-ubuntu`
+: An extension to `git` that can clone the repository attached to a package directly from Launchpad.
+It has many other features intended to enable more complex package workflows, which are described in the [Ubuntu Maintainer's Handbook](https://github.com/canonical/ubuntu-maintainers-handbook).
+
+`pull-lp-source`
+: Similar to `git-ubuntu`, which should be used instead where possible.
+The main difference is that instead of cloning the `git` repository, it grabs the source package from Launchpad.
+That means you'll get things like the orig tarball, but not the `git` history.
+
+**Rust-specific tools:**
+
+`debcargo`
+: A tool written in Rust that will attempt to automatically convert a Rust crate into a Debian source package, by mapping dependencies to Debian packages and generating important files like `debian/control`, `debian/rules`, and `debian/changelog`.
+
+`cargo-debstatus`
+: Pull the dependencies from a Cargo project, and determine if appropriate versions of them are already available in the package archive.
+
+### Tools for building the binary package from the source package
+
+**Language-agnostic tools:**
+
+`sbuild`
+: The primary tool Ubuntu Rust packagers will likely use to build binary packages.
+Fundamentally it is an orchestration tool and replaces the direct invocation of many other tools.
+It creates clean build environments using `schroot`s as a sandbox, containing all the necessary dependencies already installed.
+It then invokes `dpkg-buildpackage`, and copies the output out of the `schroot`.
+There is an alternative backend for `sbuild`, which replaces `schroot`, called `unshare`.
+At the time of writing, Launchpad still uses `schroot` and it is probably preferable to do the same locally to avoid behavior differences.
+
+`dpkg-buildpackage`
+: The lower-level build tool that is invoked for you by `sbuild`.
+It is responsible for running all of the scripts defined in `debian/rules`.
+
+`debhelper`
+: A suite of tools used to simplify many of the elements that packages often have in common.
+It defines a clear sequence of steps that packages go through when building, which can all be overridden as necessary within `/debian/rules`.
+Giving names and order to these steps means that packages no longer need custom scripts to do things like apply patches, or perform configuration.
+If you see a call to `dh` in `/debian/rules`, that package is using `debhelper`.
+Some older packages may not have migrated to `debhelper` from the manual approach yet.
+Lots of add-on packages exist, most notably `dh-cargo` which is listed separately.
+
+`lintian`
+: The static analysis tool use to check packages for policy violations, usually executed on your behalf by `sbuild`.
+While it's listed under source tooling, it can also check binary packages.
```

Comment:
Same as the preceding section: add to https://documentation.ubuntu.com/project/contributors/setup/?